README.md

```diff
@@ -204,10 +207,23 @@ Usage
 
 PuppetDB supports a large number of configuration options for both configuring the puppetdb service and connecting that service to the puppet master.
 
+### puppetdb::globals
+The `puppetdb::globals` class is intended to provide similar functionality to the `puppetlabs-postgresql`'s `postgresql::globals` class providing a top-level entry-point into the module so that we can properly set defaults for the `puppetdb::params` class based on the version of `puppetdb` you are using. This setting defaults to `present`. 
```

Comment:
"intended to provide similar functionality to the `puppetlabs-postgresql`'s `postgresql::globals` class providing "

seems a bit misworded. Maybe change to 

"intended to provide similar functionality to the `postgresql::globals` class in the `puppetlabs-postgresql` module by exposing "

or something